internal/provider/provider.go

```diff
@@ -88,7 +88,7 @@ func (p *BloxOneProvider) Configure(ctx context.Context, req provider.ConfigureR
 		option.WithAPIKey(data.APIKey.ValueString()),
 		option.WithCSPUrl(data.CSPUrl.ValueString()),
 		option.WithDefaultTags(dfTags),
-		option.WithDebug(false),
+		option.WithDebug(true),
```

Comment:
shouldn't change it. we should make it modifiable by an env variable so that it doesn't have to be enabled in code and thus end up in the PR.

Reply:
done, we will include this next time, in the PR, made it as false